tests/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(6, 2)) == 6
 
 
 def test_should_split_a_number_into_equal_parts_when_value_is_divisible_by_number_of_parts():
-    pass
+    goals = split_integer(20, 4)
+
+    for item in goals[:-1]:
```

Comment:
It's more difficult to read than the following check:
```
assert split_integer == [5, 5, 5, 5]
```
If you can write the result implicitly do it) 
It's easier to understand the test in this case

Reply:
Changes provided, in future i will try to make tests more simple.